src/pages/edge/edge-network/tutorials/datastream-config-override-rules.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Datastream config override for Edge events using Rules.
+description: This guide provides instructions on overriding datastream ID and datastream configuration for Edge Network Requests that are not initiated by the sendEvent API, all accomplished through Tags Rules.
+keywords:
+- Edge Network
+- Datastream config overrides
+- Datastream ID override
+- Guide
+---
+
+# Datastream config override for Edge events using Rules
+
+By default, all **Edge Experience events** are sent to the datastream specified in the **Adobe Experience Platform Edge Network** extension configuration settings. This tutorial offers examples on how to customize this configuration on a per-event basis
+
+<InlineAlert variant="info" slots="text"/>
+
+Starting with `Edge` extension version **2.4.0** (Android) and **4.3.0** (iOS), the `sendEvent` API supports optional datastream overrides. For detailed information, please refer to the [sendEvent API reference](./api-reference.md#sendEvent).
+
+## Steps to add datastream config overrides for Lifecycle Edge events using rules
+
+Let's add **datastream ID override** or **datastream config overrides** for `Lifecycle Edge` events of type **application.launch** and **application.close** by following the next steps.
+
+### 1. Add a new rule for your mobile property
```

Comment:
Minor nit - I don't love numbering of steps in headers since it's already very strongly implied that the steps are already in order when you're saying "... by following the next steps."

So, all this to say, I'd prefer if we removed the 1. 2. 3. from the subheaders, but if you think it's better in, you can leave them in.